### PR TITLE
Upgraded to RavenDB client 6.0.104

### PR DESCRIPTION
**Pull request by ebrob**
Upgraded packages to current. Minor update to test code because of constructor changes. All unit tests pass.

### CODE REVIEW

**Comment by ryanheath** [comment]
There are a few files changes as if many lines have changed.
What happened there?
I'd rather see only the real changes like in the other files.

Are you able to fix that?

// Ryan

File: test/Raven.Client.NodaTime.Tests/NodaDurationTests.cs
Comment (ryanheath) [inline test/Raven.Client.NodaTime.Tests/NodaDurationTests.cs:1]: Please only change the lines that need a change.

File: test/Raven.Client.NodaTime.Tests/NodaLocalDateTimeTests.cs
Comment (ryanheath) [inline test/Raven.Client.NodaTime.Tests/NodaLocalDateTimeTests.cs:1]: Please only change the lines that need a change.

File: test/Raven.Client.NodaTime.Tests/NodaZonedDateTimeTests.cs
Comment (ryanheath) [inline test/Raven.Client.NodaTime.Tests/NodaZonedDateTimeTests.cs:1]: Please only change the lines that need a change.

File: src/Raven.Bundles.NodaTime/Raven.Bundles.NodaTime.csproj
Comment (ryanheath) [inline src/Raven.Bundles.NodaTime/Raven.Bundles.NodaTime.csproj:1]: bump the version to <Version>6.0.104</Version>

File: src/Raven.Client.NodaTime/Raven.Client.NodaTime.csproj
Comment (ryanheath) [inline src/Raven.Client.NodaTime/Raven.Client.NodaTime.csproj:1]: bump the version to <Version>6.0.104</Version>

File: test/Raven.Client.NodaTime.Tests/NodaDurationTests.cs
Comment (ebrob) [reply test/Raven.Client.NodaTime.Tests/NodaDurationTests.cs:1]: The constructor for `GetDocumentsCommand` changed, so in order to get the unit tests working I had to update that code. For whatever reason, they added `DocumentConventions` as a required argument. See:

https://github.com/ravendb/ravendb/blob/v6.0/src/Raven.Client/Documents/Commands/GetDocumentsCommand.cs

Comment (ebrob) [reply test/Raven.Client.NodaTime.Tests/NodaDurationTests.cs:1]: BTW, thank you for this library! We have been using it in our project  for a number of years now.

Comment (ebrob) [reply test/Raven.Client.NodaTime.Tests/NodaDurationTests.cs:1]: Oh, I think I see what you mean -- for some reason it's showing a huge number of changes. Very weird. Let me take a look.

Comment (ryanheath) [reply test/Raven.Client.NodaTime.Tests/NodaDurationTests.cs:1]: 👍

**Pull request closed without merging**